src/XMLexport.cpp

```diff
@@ -445,11 +445,11 @@ void XMLexport::writeHost(Host* pHost, pugi::xml_node mudletPackage)
     QColor outerColor;
     QColor innerColor;
     pHost->getPlayerRoomStyleDetails(styleCode, outerDiameterPercentage, innerDiameterPercentage, outerColor, innerColor);
-    host.append_attribute("playerRoomPrimaryColor") = outerColor.name(QColor::HexArgb).toLatin1().constData();
-    host.append_attribute("playerRoomSecondaryColor") = innerColor.name(QColor::HexArgb).toLatin1().constData();
-    host.append_attribute("playerRoomStyle") = QString::number(styleCode).toLatin1().constData();
-    host.append_attribute("playerRoomOuterDiameter") = QString::number(outerDiameterPercentage).toLatin1().constData();
-    host.append_attribute("playerRoomInnerDiameter") = QString::number(innerDiameterPercentage).toLatin1().constData();
+    host.append_attribute("playerRoomPrimaryColor") = outerColor.name(QColor::HexArgb).toUtf8().constData();
```

Comment:
As it happens the texts here are all strictly ASCII in encoding - the forms for the names used in the first two can be checked out at https://doc.qt.io/qt-5/qcolor.html#setNamedColor and the last three are all arabic ('0' to '9') numbers as strings...

Reply:
We generate XML here, thus we encode *to* UTF-8. Whether or not the source is (supposed to be) limited to a subset that would result in an identical result when encoded with a different primitive doesn't matter IMHO.

As the destination is definitely not latin1-encoded, I propose that we pick the conversion method that's least-wrong. There's not going to be a measurable sped difference either.